test/clientOptions.test.ts

```diff
@@ -98,7 +98,28 @@ describe("custom client options", function () {
 
     // Note:
     // core-rest-pipeline skips the retry throwing `RestError: getaddrinfo ENOTFOUND azure.azconfig.io`
-    // Probably would be fixed in upstream libs.
     // See https://github.com/Azure/azure-sdk-for-js/issues/27037
-    it("should retry on DNS failure");
+    // Fixed in @azure/core-rest-pipeline@1.2.2

```

Comment:
Can we remove the comments now that its fixed?

Reply:
sure, removed.